tools/flang1/flang1exe/func.c

```diff
@@ -3967,14 +3967,22 @@ rewrite_calls(void)
         }
         sptr_lhs = memsym_of_ast(A_SRCG(ast));
         if (allocatable_member(sptr_lhs)) {
-          rewrite_deallocate(A_SRCG(ast), false, std);
-          if (!ALLOCG(sptr_lhs) && !ALLOCATTRG(sptr_lhs) &&
-              !POINTERG(sptr_lhs)) {
-            /* Has allocatable members but item itself is not
-             * allocatable nor pointer
-             */
+          /* Has allocatable members but item itself is not
+           * allocatable nor pointer
+           */
+          bool lhs_not_allocatable =
+            !ALLOCG(sptr_lhs) && !ALLOCATTRG(sptr_lhs) && !POINTERG(sptr_lhs);
+          /* HCCSYMG() checks if sptr_lhs is a temp variable or not. 
+           * It is used inside rewrite_allocatable_assignment() fuction 
```

Comment:
Ooops! Just spotted another typo... function not fuction.  ;-)

Reply:
We need to run a spellchecker on our patches... 😆